### ISSUE TITLE
Errors in dnn() examples: invalid loss arguments

### ISSUE THREAD

**Issue by tripartio** [citoverse/cito#117]
Hello. The following lines from the [`dnn()`](https://citoverse.github.io/cito/reference/dnn.html) examples throw errors:

```r
## Multinomial and conditional logit regression
m = dnn(Species~., data = iris, loss = "clogit", lr = 0.01)
# Error in match.arg(loss) :    'arg' should be one of “mse”, “mae”, “softmax”, “cross-entropy”, “gaussian”, “binomial”, “poisson”, “mvp”, “nbinom”

m = dnn(Species~., data = iris, loss = "multinomial", lr = 0.01)
# Error in match.arg(loss) :    'arg' should be one of “mse”, “mae”, “softmax”, “cross-entropy”, “gaussian”, “binomial”, “poisson”, “mvp”, “nbinom”

Y = t(stats::rmultinom(100, 10, prob = c(0.2, 0.2, 0.5)))
m = dnn(cbind(X1, X2, X3)~., data = data.frame(Y, A = as.factor(runif(100))), loss = "multinomial", lr = 0.01)
# Error in match.arg(loss) :    'arg' should be one of “mse”, “mae”, “softmax”, “cross-entropy”, “gaussian”, “binomial”, “poisson”, “mvp”, “nbinom”

```

Perhaps there's been a change in the names of appropriate loss functions? Have "clogit" and "multinomial" been renamed "softmax" and "cross-entropy" or something like that?
